types/src/blob/msg_pay_for_blobs.rs

```diff
@@ -80,7 +81,7 @@ impl From<MsgPayForBlobs> for RawMsgPayForBlobs {
         let share_commitments = msg
             .share_commitments
             .into_iter()
-            .map(|c| c.0.to_vec())
+            .map(|c| Hash::from(c).to_vec())
```

Comment:
```suggestion
            .map(|c| c.hash().to_vec())
```

very minor thing but i prefer this